app/views/teachers/EditStudentModal.coffee

```diff
@@ -0,0 +1,39 @@
+ModalView = require 'views/core/ModalView'
+State = require 'models/State'
+template = require 'templates/teachers/edit-student-modal'
+
+module.exports = class EditStudentModal extends ModalView
+  id: 'edit-student-modal'
+  template: template
+
+  events:
+    'click .send-recovery-email-btn:not(.disabled)': 'onClickSendRecoveryEmail'
+    'click .change-password-btn:not(.disabled)': 'onClickChangePassword'
+    'input .new-password-input': 'onChangeNewPasswordInput'
+
+  initialize: ({ @user }) ->
+    @supermodel.trackRequest @user.fetch()
+    @utils = require 'core/utils'
+    @state = new State({
+      emailSent: false
+      passwordChanged: false
+      newPassword: ""
+    })
+    @listenTo @state, 'change', @render
+
+  onClickSendRecoveryEmail: ->
+    email = @user.get('email')
+    res = $.post '/auth/reset', {email: email}, =>
+      @state.set { emailSent: true }
```

Comment:
Since this isn't a `/db/*` endpoint, it wouldn't make sense to have this on a Model or Collection, but it would still be good to keep url construction out of views and endpoint calling uniform. Make a `sendRecoveryEmail` function in `/app/core/auth.coffee` with the same parameter structure as other endpoint methods: `(param1, param2..., options)`, so in this case `(email, options)`.